mitxpro/urls.py

```diff
@@ -104,7 +105,7 @@
                 name="wagtailimages_serve",
             )
         ],
-        cache_control(max_age=WAGTAIL_IMG_CACHE_AGE),
+        cache_control_max_age_jitter(cache_control(max_age=WAGTAIL_IMG_CACHE_AGE)),
```

Comment:
So I think the issue you're hitting here is probably that this needs to be:

```suggestion
        cache_control_max_age_jitter()(cache_control(max_age=WAGTAIL_IMG_CACHE_AGE)),
```

If that doesn't work, or becaue it's simplier, I think you could do this: 

```suggestion
        cache_control_max_age_jitter(max_age=WAGTAIL_IMG_CACHE_AGE),
```

You still probably need the change from https://github.com/mitodl/ol-django/pull/74

Reply:
sure

Reply:
first approach `cache_control_max_age_jitter()(cache_control(max_age=WAGTAIL_IMG_CACHE_AGE)),` gave the following error

Traceback (most recent call last):

  File "manage.py", line 19, in <module>

    execute_from_command_line(sys.argv)

  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line

    utility.execute()

  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 413, in execute

    self.fetch_command(subcommand).run_from_argv(self.argv)

  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 354, in run_from_argv

    self.execute(*args, **cmd_options)

  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 398, in execute

    output = self.handle(*args, **options)

  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 89, in wrapped

    res = handle_func(*args, **kwargs)

  File "/usr/local/lib/python3.7/site-packages/django/core/management/commands/migrate.py", line 75, in handle

    self.check(databases=[database])

  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 423, in check

    databases=databases,

  File "/usr/local/lib/python3.7/site-packages/django/core/checks/registry.py", line 76, in run_checks

    new_errors = check(app_configs=app_configs, databases=databases)

  File "/usr/local/lib/python3.7/site-packages/django/core/checks/urls.py", line 13, in check_url_config

    return check_resolver(resolver)

  File "/usr/local/lib/python3.7/site-packages/django/core/checks/urls.py", line 23, in check_resolver

    return check_method()

  File "/usr/local/lib/python3.7/site-packages/django/urls/resolvers.py", line 416, in check

    for pattern in self.url_patterns:

  File "/usr/local/lib/python3.7/site-packages/django/utils/functional.py", line 48, in __get__

    res = instance.__dict__[self.name] = self.func(instance)

  File "/usr/local/lib/python3.7/site-packages/django/urls/resolvers.py", line 602, in url_patterns

    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)

  File "/usr/local/lib/python3.7/site-packages/django/utils/functional.py", line 48, in __get__

    res = instance.__dict__[self.name] = self.func(instance)

  File "/usr/local/lib/python3.7/site-packages/django/urls/resolvers.py", line 595, in urlconf_module

    return import_module(self.urlconf_name)

  File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module

    return _bootstrap._gcd_import(name[level:], package, level)

  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import

  File "<frozen importlib._bootstrap>", line 983, in _find_and_load

  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked

  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked

  File "<frozen importlib._bootstrap_external>", line 728, in exec_module

  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "/src/mitxpro/urls.py", line 116, in <module>

    + (

  File "/usr/local/lib/python3.7/site-packages/wagtail/utils/urlpatterns.py", line 13, in decorate_urlpatterns

    pattern.callback = update_wrapper(decorator(pattern.callback), pattern.callback)

  File "/home/mitodl/.local/lib/python3.7/site-packages/mitol/common/decorators.py", line 15, in _cache_controlled

    "cache_control_max_age_jitter didn't receive an HttpRequest. If you are "

TypeError: cache_control_max_age_jitter didn't receive an HttpRequest. If you are decorating a classmethod, be sure to use @method_decorator.